src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/HelpWithFeesDetails.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.civil.model.citizenui;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.reform.civil.enums.FeeType;
+import uk.gov.hmcts.reform.civil.enums.HWFFeeDetailsSummary;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class HelpWithFeesDetails {
+
+    private String noRemissionDetails;
+    private HWFFeeDetailsSummary noRemissionDetailsSummary;
```

Comment:
HWFFeeDetailsSummary This enum class is specific to noRemission so can you change the name accordingly?

Reply:
changed to noRemissionDetailsSummary